lib/runtime/wasmer/exports.go

```diff
@@ -405,6 +407,30 @@ func (in *Instance) ParachainHostValidationCode(parachaidID uint32, assumption p
 	return validationCode, nil
 }
 
+// ParachainHostValidationCodeByHash returns validation code for the given hash.
+func (in *Instance) ParachainHostValidationCodeByHash(validationCodeHash common.Hash) (
```

Comment:
I see that the same method is implemented in the `wazero` package, does make sense to keep this method here as we are about to remove the `wasmer` from the project?

Reply:
Yeah, looks like we need to rebase. @kishansagathiya could you help?

Reply:
I copied all the parachain related tests to `wazero`. I'll keep the changes in the `wasmer` for now. It'll get removed in the rebase anyway.

Reply:
Thank you!